spec/apollo-federation/entities_field_spec.rb

```diff
@@ -475,6 +475,59 @@ def load_entity
                   end
                 end
               end
+
+              context 'when reference keys have multiple words' do
```

Comment:
Can we add one more test covering the inheritance of `underscore_reference_keys`, such that it it enabled in the base object and validates that it still works in a child object class?